### PR TITLE
Add file with image installation instructions.

### PR DESCRIPTION
**Pull request by jenninikko**
Signed-off-by: Jenni Nikolaenko <evgeniia.nikolaenko@unikie.com>

### CODE REVIEW

**Comment by jenninikko** [comment]
@vilvo @brianmcgillion

File: README.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+The canonical URL for the upstream Spectrum git repository is
+<https://spectrum-os.org/git/spectrum/>.
+
+To try Spectrum OS, you need to build it from source and then install it to an SD card.
+
+> Cross-compiled version of Spectrum OS is for i.MX 8QXP board only.
```
Comment (vilvo) [inline README.md:6]: As of now, cross-compiled version of ...

File: README.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+## Building a Spectrum OS Image
+
+Clone sources of spec and nix-spec (cross-compile branches for i.MX 8QXP):
+
+	$ git clone -b aarch64-imx8-crosscompile https://github.com/tiiuae/nixpkgs-spectrum/
```
Comment (vilvo) [inline README.md:12]: Please align indentations

File: README.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    $ git clone -b aarch64-imx8-crosscompile https://github.com/tiiuae/spectrum/
+
+Set up the Spectrum binary cache.
```
Comment (vilvo) [inline README.md:15]: Please add - "Using binary caches speeds up builds significantly."
It helps unfamiliar user to understand why.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
+The canonical URL for the upstream Spectrum git repository is
+<https://spectrum-os.org/git/spectrum/>.
+
+To try Spectrum OS, you need to build it from source and then install it to an SD card.
```
Comment (vilvo) [inline README.md:4]: Please add here options to "To try Spectrum OS":
a. download binary image (link to arm-kal.us.to - let's change the link to vedenemo.dev hydra cross-compilation later)
b. "build it from source..."

File: README.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+## Installing Spectrum OS
+
+Before installation:
+
+* Prepare a 4 GB SD card.
+* Make sure that the i.MX 8QXP board is configured to boot from an SD card.
+
+To run Spectrum OS on i.MX 8QXP board, perform the following steps:
+
+* Use the image from the previous step or download the prebuilt image _20220826-spectrum-live-imx8qxp-yuriy-custom.img_ from the page: <http://arm-kal.us.to:20080/>.
```
Comment (vilvo) [inline README.md:60]: Please list downloading pre-built image in the beginning of the page.

File: README.md [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+	Note the following:
+
+	- The custom binary cache is located here: <http://binarycache.vedenemo.dev>.
+	- The public key of this cache is:
+      `binarycache.vedenemo.dev:Yclq5TKpx2vK7WVugbdP0jpln0/dPHrbUYfsH3UXIps=`
+    - Spectrum OS own binary cache is located here: <https://cache.dataaturservice.se/spectrum/>.
+    - The public key of this cache is:
+        `spectrum-os.org-1:rnnSumz3+Dbs5uewPlwZSTP0k3g/5SRG4hD7Wbr9YuQ=`
+
+    All configurations are made in the `/etc/nix/nix.conf` file. It should be possible to carry everything to the NixOS configuration with little effort.
```
Comment (vilvo) [inline README.md:28]: Proposal - let's simplify this chapter by listing only:
1. vedenemo binary-cache details and link to Spectrum binary cache https://spectrum-os.org/doc/binary-cache.html
2. Link to Binary Cache usage - https://nixos.wiki/wiki/Binary_Cache#Using_a_binary_cache 

More detailed comments why this is chapter as of now is problematic:

- it only covers non-NixOS based setup and does not cover NixOS based setups
- it replicates info already available from Spectrum and NixOS - it's easier for us to keep info up to date by not replicating

File: README.md [hunk 7]
```diff
@@ -0,0 +1,66 @@
+The canonical URL for the upstream Spectrum git repository is
+<https://spectrum-os.org/git/spectrum/>.
+
+To try Spectrum OS, you need to build it from source and then install it to an SD card.
+
+> Cross-compiled version of Spectrum OS is for i.MX 8QXP board only.
+
+## Building a Spectrum OS Image
```
Comment (vilvo) [inline README.md:8]: Please refactor "## Setting up binary caches (optional, recommended)"-chapter and put it before ## Building a Spectrum OS Image

File: README.md [hunk 8]
```diff
@@ -0,0 +1,66 @@
...
+* Prepare a 4 GB SD card.
+* Make sure that the i.MX 8QXP board is configured to boot from an SD card.
+
+To run Spectrum OS on i.MX 8QXP board, perform the following steps:
+
+* Use the image from the previous step or download the prebuilt image _20220826-spectrum-live-imx8qxp-yuriy-custom.img_ from the page: <http://arm-kal.us.to:20080/>.
+
+* Copy the downloaded image to your SD card to create bootable media:
+
+	  $ sudo dd if=spectrum-live.img of=/dev/sdx bs=1M conv=fsync
```
Comment (vilvo) [inline README.md:64]: Classic disclaimer must be put here. "WARNING! Always check your SD-card device and replace `/dev/sdx` with your SD. Check the device from kernel log after inserting the SD card to your computer. Using wrong device can make your computer not boot."

**Review by vilvo — CHANGES_REQUESTED** [review]
Good work, key info in place. Please address review comments and let's merge this.

**Comment by vilvo** [comment]
> Good work, key info in place. Please address review comments and let's merge this.

Please add link to https://nixos.org/download.html#download-nix as "Requirement" under ## Building a Spectrum OS Image

**Comment by vilvo** [comment]
Merging as baseline doc. Proposed review comments for later.

File: README.md (hunk 1 above)
Comment (jenninikko) [reply README.md:6]: Changed to: 
Currently, the cross-compiled version of Spectrum OS is for i.MX 8QXP board only.

File: README.md (hunk 2 above)
Comment (jenninikko) [reply README.md:12]: +

File: README.md (hunk 3 above)
Comment (jenninikko) [reply README.md:15]: Changed to:
Set up the Spectrum binary cache to speed up the build process:...

File: README.md (hunk 4 above)
Comment (jenninikko) [reply README.md:4]: Done.

File: README.md (hunk 5 above)
Comment (jenninikko) [reply README.md:60]: Done.

File: README.md (hunk 8 above)
Comment (jenninikko) [reply README.md:64]: Done.

File: README.md (hunk 7 above)
Comment (jenninikko) [reply README.md:8]: Done.

File: README.md (hunk 6 above)
Comment (jenninikko) [reply README.md:28]: Made some attempts. If something comes up, lets' discuss this.

**Pull request merged**